spec/features/user_spec.rb

```diff
@@ -69,4 +80,19 @@
     register_user
     expect(page).to have_content('Добро пожаловать!')
   end
+
+  it 'not allows to create user with wrong data' do
+    visit sign_up_path
+    click_button 'Регистрация'
+    expect(page).to have_content('Заполните все поля!')
+  end
+
+  it 'not allows to create user with already existing name or email' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.